File: PYTHON/Kode Program/person.py
# kelas Person sebagai kelas dasar
class Person:
    def __init__(self, nama="", umur=0, gender="", alamat=""):
        self.__nama = nama
        self.__umur = umur
        self.__gender = gender
        self.__alamat = alamat

    # Getter & Setter
    def get_nama(self): return self.__nama
    def set_nama(self, nama): self.__nama = nama

    def get_umur(self): return self.__umur
    def set_umur(self, umur): self.__umur = umur

    def get_gender(self): return self.__gender
    def set_gender(self, gender): self.__gender = gender

    def get_alamat(self): return self.__alamat
    def set_alamat(self, alamat): self.__alamat = alamat

File: JAVA/Kode Program/Person.java
// kelas Person sebagai kelas induk
public class Person {
    protected String nama;
    protected int umur;
    protected String gender;
    protected String alamat;

    // konstruktor
    public Person(String nama, int umur, String gender, String alamat) {
        this.nama = nama;
        this.umur = umur;
        this.gender = gender;
        this.alamat = alamat;
    }

    // method untuk menampilkan informasi dasar
    public void tampilkanInfo() {
        System.out.printf("%-15s %-5d %-12s %-15s", nama, umur, gender, alamat);
    }
}
